libzmix/src/errors.rs

```diff
@@ -0,0 +1,149 @@
+use commitments::pok_vc::PoKVCError;
+use failure::{Backtrace, Context, Fail};
+
+pub mod prelude {
+    pub use super::{BBSError, BBSErrorExt, BBSErrorKind, PSError};
+}
+
+#[derive(Debug, Fail)]
+pub enum PSError {
+    #[fail(
+        display = "Verkey has unequal number of Y and Y_tilde elements. Y={} and Y_tilde={}",
```

Comment:
This error may mean something to those intimately familiar with the PS paper, but it doesn't convey much to the average user.